bindings/wysiwyg-wasm/src/lib.rs

```diff
@@ -77,6 +77,12 @@ pub struct ComposerModel {
     inner: wysiwyg::ComposerModel<Utf16String>,
 }
 
+impl Default for ComposerModel {
+    fn default() -> Self {
+        Self::new()
+    }
+}
+
```

Comment:
I don't think we need these default methods? In fact, I think we should never have an empty `ComposerModel`.

Reply:
I know... the thing is that clippy complains that if you have an empty new implementation like we do, we either remove that implementation or we create also the Default impl for it.
Can either remove both this and the `new()` if we never use it or keep this

Reply:
I don't think we enforce clippy warnings on the repo at the moment, but maybe then it's best to rename the `new` function to `default`, as that seems to be the Rust way. Is that right @andybalaam ?

Reply:
you should use default like this
`let mut model: ComposerModel = Default::default()`
in place of using the new, of course a struct contains sub structs that implement default, you can also use `#derive(Default)`

Reply:
You know what? Let's keep the warning for now, I'll remove the default implementation.
The cleanest thing to do would be to go through all our structs, make them implement default through a derive, so that we can actually have more complex structs derive from default and use it in place of the empty `new()`

But yeah all this work is worth doing into another PR

Reply:
Yeah, implementing or deriving `Default` is good when there is a sensible default/empty value for an object.

No, we don't enforce clippy and we should!